packages/cloud-cognitive/src/components/AddSelect/AddSelectList.js

```diff
@@ -85,31 +90,51 @@ export let AddSelectList = ({
     setAppliedModifiers(modifiersClone);
   };
 
+  const metaPanelHandler = (item) => {
+    if (item.meta) {
+      setDisplayMetaPanel(item);
+    }
+  };
+
+  const isInMetaPanel = (id) => id === displayMetalPanel?.id;
+
   return (
-    <div className={`${blockClass}-wrapper`}>
+    <div
+      className={cx(`${blockClass}-wrapper`, {
+        [`${blockClass}-wrapper-multi`]: multi,
+      })}
+    >
       <StructuredListWrapper selection className={`${blockClass}`}>
         <StructuredListBody>
           {filteredItems.map((item) => (
             <StructuredListRow
               key={item.id}
               className={cx(`${blockClass}-row`, {
                 [`${blockClass}-row-selected`]: isSelected(item.id),
+                [`${blockClass}-row-meta-selected`]: isInMetaPanel(item.id),
               })}
+              onClick={(evt) => metaPanelHandler(item, evt)}
             >
-              <StructuredListCell className={`${blockClass}-cell`}>
+              <div className={`${blockClass}-cell`}>
                 <div className={`${blockClass}-cell-wrapper`}>
                   {multi ? (
                     <>
                       <div className={`${blockClass}-checkbox`}>
-                        <Checkbox
-                          onChange={(checked) =>
-                            handleMultiSelection(item.id, checked)
-                          }
-                          labelText={item.title}
-                          id={item.id}
-                          checked={isSelected(item.id)}
-                          className={`${blockClass}-checkbox-wrapper`}
-                        />
+                        {
+                          // hacky way to prevent checkbox from triggering the meta onclick handler
```

Comment:
Can you call `event.stopPropagation()` in the checkbox onChange event? Or does that not prevent the click from bubbling up?

Reply:
Maybe something like this:
```jsx
onChange={(value, id, event) => {
    event.stopPropagation();
    handleMultiSelection(value, id);
}}
```